jsons/styles.json

```diff
@@ -516,5 +516,13 @@
       "1.0.0"
     ],
     "reference": "Vlasov et al. (2021) "
+  },
+  {
+    "style": "csv_filter_style_1",
+    "name": "csv_filter_style_1",
+    "versions": [
+      "1.0.0"
+    ],
+    "reference": "Vlasov et al. (2021) "
```

Comment:
Nice :) ... but I wrote it! :)

Reply:
Ursgal team 2021?

Reply:
sry just copied it from the Lima style above xD should also have been simply Ursgal Team ^^